connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -1566,6 +1566,25 @@ class ClientE2ETestSuite
     val result = df.select(trim(col("col"), " ").as("trimmed_col")).collect()
     assert(result sameElements Array(Row("a"), Row("b"), Row("c")))
   }
+
+  test("SPARK-49673: new batch size, multiple batches") {
+    val maxBatchSize = spark.conf.get("spark.connect.grpc.arrow.maxBatchSize").dropRight(1).toInt
```

Comment:
This drops the limit by one order of magnitude. Could we run this at 80% of the limit?

Reply:
Sorry not sure what you mean here. I've decreased both the GRPC_MAX_MESSAGE_SIZE in the client and the maxBatchSize on the server to 10MiB to simulate the batches being close to the message size limit without having to process `range(128*1024*1024)` on the cluster. Are you asking me to replace this with spark.connect.grpc.arrow.maxBatchSize = spark.connect.grpc.arrow.maxBatchSize = CONNECT_GRPC_MAX_MESSAGE_SIZE = 0.8 * 128MiB?

Reply:
Ah, I think there might be a misunderstanding here. The dropRight is used to drop the "b" character from the "1269760b" String returned by the spark.conf.get("spark.connect.grpc.arrow.maxBatchSize") call 😄

Reply:
Ah, never mind then :)